environs/bootstrap.go

```diff
@@ -61,6 +61,14 @@ type BootstrapParams struct {
 	// that rely on it for selecting images. This will be empty for
 	// providers that do not implements simplestreams.HasRegion.
 	ImageMetadata []*imagemetadata.ImageMetadata
+
+	// JujuDbSnapPath is the path to a local .snap file that will be used
+	// to run the juju-db service.
+	JujuDbSnapPath string
```

Comment:
Just checking, are these used? The CLI args to bootstrap.BootstrapParams. DO we need this on environs.BootstrapParams also?